src/pages/EmployeeAccount.vue

```diff
@@ -0,0 +1,23 @@
+<script>
+import { defineComponent } from "vue";
+import { mapActions } from "pinia";
+import { useAuthStore } from "@/stores/auth";
+
+export default defineComponent({
+  name: "EmployeeAccount",
+  methods: {
+    ...mapActions(useAuthStore, ["restoreUserData"]),
+  },
+  created() {
+    this.restoreUserData();
```

Comment:
I'm not sure if I know what you mean, so far I ensured that user cannot visit "/employee" an employee cannot visit "/user". When it comes other pages and handling redirections depending on the role of user - it will be handled in the next pull requests.

Reply:
Sorry, I will try to be more specific. What if the employee is logged in, but is on another page and refreshes the browser? With this logic, it seems like the employee won't be able to enter (reason: redirections depending on the role of the user you mentioned) this page because the data from storage won't be restored. What do you think?

Reply:
Pages: homepage, movies, movie, screenings, contact, logi and register are public, only the header changes depending on the fact if the suer is logged in. Handling the behaviour for employee only will be analysed in details later on. When a user-user or user-employee are logged in and I refresh the page I don't loose the state, I still have info what kind of user is logged in.

Reply:
That is what I thought will happen after a page refresh (losing info). If you still have info about the user, is `restoreUserData` needed here?